### PR TITLE
[WIP] Blockouts UI

### PR DESCRIPTION
**Pull request by benjaminwood**
This PR is already larger than I intended. I'm opening it now as a work-in-progress. I'm expecting to wrap it up today. I'll assign reviewers when it is ready!

### CODE REVIEW

**Comment by benjaminwood** [comment]
This is on staging now! http://omd-scheduler.herokuapp.com

@kyleboe, @nvick & @SarahOMD can you try it out on staging and give me your feedback? @SarahOMD if you would benefit from a demo/walk-through, let me know.

We still have some styling work to do, but that can happen after this PR is accepted.

There are a couple known/minor issues that I haven't decided how to deal with yet. I'm going to have @nshoes review code tomorrow and help refactor/clean things up before we merge this.

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@ class ApplicationController < ActionController::Base
   include Pundit
   before_action :authenticate_user!
   before_action :configure_permitted_parameters, if: :devise_controller?
+  # around_action :set_time_zone
```
Comment (kyleboe) [inline app/controllers/application_controller.rb]: Does this need to be uncommented?

Comment (SarahOMD) [reply app/controllers/application_controller.rb]: Looks beautiful!  Just a couple thoughts as I poked around in there... 
1) We only operate during office hours (8am - 5pm), so there's really no need to list all hours of the day.  I'm afraid our less intuitive folks might feel like they need to set a Do Not Disturb time frame for each day of the year.  
2) There are two boxes for an "End Date".  I wasn't sure which one to use, as they both seem to do the same thing.

Comment (benjaminwood) [reply app/controllers/application_controller.rb]: @kyleboe yes, thank you! I'll do that.

@SarahOMD thanks for trying it out and providing your feedback!

1. I'll change the time dropdowns to 8am - 5pm as you suggested!
2. Can you clarify what you mean about the second 'End Date' box? There should be a 'Start Date' & 'End Date'. Additionally, if you configure a recurring blockout and change 'forever' to 'until' another date selection box appears where you can configure a date in the future where the recurrence ends. So for example, I could setup a blockout for where the start date is Wednesday, the end date is Friday, and it recurs every week until the end of next month.

Comment (benjaminwood) [reply app/controllers/application_controller.rb]: @kyleboe I've enable the `set_time_zone` around_action.

@SarahOMD I've updated the time dropdowns to 8am - 5pm and mentioned above :point_up: 

It's on staging now: http://omd-scheduler.herokuapp.com

Comment (SarahOMD) [reply app/controllers/application_controller.rb]: It looks great!!  Not confusing anymore :-) . Thanks!

**Pull request merged**